elections/arch-committee-2020-09/README.md

```diff
@@ -0,0 +1,15 @@
+# Architecture Committee Elections: February 2020
```

Comment:
One sec - shouldn't this be "September 2020"?

Reply:
@jodh-intel  Nice caught. 
Updated.